src/scripts/fetch/fetch.ts

```diff
@@ -0,0 +1,58 @@
+interface Fetchparams {
+	param: string;
+	value: any;
+}
+
+/**
+ * Fetches data from themoviedb.org API endpoint and returns to the user
+ *
+ * @param URLExtension - the extension over the base API for example: /movie/550
+ * @param APIkey - your api key
+ * @param URLparams array of url parameters to add to fetch request. For example [{param:Query, value:string}]
+ * @returns Returns base object from the API parsed through response.json
+ */
+export const APIFetch = async (
+	URLExtension: string,
+	APIkey: string,
+	URLparams?: Fetchparams[]
+) => {
+	//Basic Fetch URL without params
+	const url = `https://api.themoviedb.org/3${URLExtension}?api_key=${APIkey}`;

```

Comment:
May be, we could just put all url's in .env as well like https://api.themoviedb.org/

Reply:
I figured leaving the extension as a param allowed us to use the fetch for any api call without worry.

You could use the stubs as env variables if you wanted a little more DRY work. but I figured at least keeping the extrension params open just saves us having to adjust the fetch API and our .env whenever we use a new endpoint. Especially as .env keys aren't shared on git and might lead to disparities.